src/Customer.php

```diff
@@ -0,0 +1,66 @@
+<?php
+namespace Clang;
+
+class Customer {
+  public $accounts; // ArrayOfAccount
```

Comment:
It would be nice to add doc-blocks for this specific class, since we use these properties in our applications.

No worries, we don't need type-hints / doc-blocks for the other Clang classes ;-)

```suggestion
  /** @var array $accounts */
  public $accounts; // ArrayOfAccount
```